### PR TITLE
Do not request duplicate comment IDs

### PR DESCRIPTION
**Pull request by robertberry-zz**

### CODE REVIEW

File: static/src/javascripts/projects/common/modules/discussion/comment-count.js [hunk 1]
```diff
@@ -31,9 +31,9 @@ define([
         defaultTemplate = commentCountTemplate;
 
     function getContentIds() {
-        return _.map(qwery('[' + attributeName + ']'), function (el) {
+        return _.uniq(_.map(qwery('[' + attributeName + ']'), function (el) {
```
Comment (gklopper) [inline static/src/javascripts/projects/common/modules/discussion/comment-count.js]: I was about to ask where the hell `.uniq(...)` came from when I noticed it was not Scala... Think I am tired :sleeping:

**Comment by gklopper** [comment]
:+1: I assume there is nothing you need do differently with the return value (i.e. it does not depend on the exact ordering).

**Comment by phamann** [comment]
:+1:

**Comment by robertberry** [comment]
@gklopper Naw. I don't think discussion does any caching based on url. If they do we should sort the keys to lower cache fragmentation (although it's never gonna help that much ... )

**Pull request merged**